.envrc

```diff
@@ -165,13 +158,6 @@ else
         die "For some reason, the virtualenv isn't Python 2.7."
 fi
 
-if [ "$(command -v sentry)" != "${PWD}/${venv_name}/bin/sentry" ]; then
```

Comment:
Instead of removing this, could we just use a less strict check (virtualenv's successfully activated at this time):

`if ! command -v sentry; then`

Reply:
:+1: